syz-runner/runner.go

```diff
@@ -119,19 +127,18 @@ func (rn *Runner) Run(firstProg []byte, idx, runIdx int) {
 		}
 
 		a := &rpctype.NextExchangeArgs{
-			Pool:    rn.pool,
-			VM:      rn.vm,
-			ProgIdx: pIdx,
-			Hanged:  hanged,
-			Info:    *info,
-			RunIdx:  rIdx,
+			Pool:       rn.pool,
+			VM:         rn.vm,
+			Hanged:     hanged,
+			Info:       *info,
+			ExecTaskID: id,
```

Comment:
All changes in the runner look like they can be moved to a separate commit. I'm not sure if there is a change in communication protocol or not. But either way it does not seem to be related to the main changes in syz-verifier.